packages/go/analysis/azure/role.go

```diff
@@ -142,6 +143,21 @@ func (s RoleAssignments) UsersWithRolesExclusive(roleTemplateIDs ...string) card
 	return result
 }
 
+func (s RoleAssignments) UsersWithRoleAssignableGroupMembership() cardinality.Duplex[uint64] {
+	members := cardinality.NewBitmap64()
+
+	// loop through all the groups
+	for groupID, groupMemberIDBitmap := range s.GroupMembership {
+		group := s.Principals.Get(azure.Group)[groupID]
+		// if that group is role assignable, set the bits
+		if isRoleAssignable, err := group.Properties.Get(azure.IsAssignableToRole.String()).Bool(); err == nil && isRoleAssignable {
```

Comment:
Do we think it would be a good idea to handle the error here? I know we can't extract information, but it may be good to at least log a warning that we couldn't convert to a Boolean value